website/docs/tutorial/1a-setting-up-bigquery.md

```diff
@@ -1,11 +1,19 @@
 ---
-title: "Setting up BigQuery for dbt Cloud"
+title: "Set up and connect BigQuery"
 id: setting-up-bigquery
-description: "Set up BigQuery with sample data and connect to dbt Cloud."
-sidebar_label: "Setting up with BigQuery"
+description: "Set up BigQuery with sample data and connect to dbt Cloud or dbt CLI."
```

Comment:
I know folks call it dbt CLI but our product is dbt Core

Reply:
Great callout. I will edit for this.